internal/cmd/project/list/list.go

```diff
@@ -44,7 +44,7 @@ func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List STACKIT projects",
-		Long:  "List all STACKIT projects that match certain criteria. At least one of parent-id, project-id-like or member flag must be provided",
+		Long:  "List all STACKIT projects that match certain criteria. At least one of parent-id, project-id-like or member flag must be provided.",
```

Comment:
```suggestion
		Long:  "List all STACKIT projects that match certain criteria.",
```
Not sure if this belongs on the long description, but I guess it should be somewhere